packages/components/src/PrimaryToolbar/__snapshots__/PrimaryToolbar.test.js.snap

```diff
@@ -126,7 +124,6 @@ exports[`PrimaryToolbar should render with data full config 1`] = `
           }
           onSelect={[MockFunction]}
           ouiaId="BulkSelect"
-          ouiaSafe={true}
```

Comment:
Some default props are missing. I think `ouiaSafe` could cause issues.

Reply:
They are missing because `defaultProps` were removed and are now assigned in the function arguments section: https://github.com/RedHatInsights/frontend-components/pull/1356/files#diff-b34328198bb4bf1ec18a03ba5562054868ca058c0e89aeb730fd0c47cb7b5ffcR47